### PR TITLE
Restrict create/update of subscription to admin only

### PR DESCRIPTION
**Pull request by rhyw**
The retrieve/list is still accessible to authenticated user.

### CODE REVIEW

**Comment by rhyw** [comment]
With this patch, we'll need to make sure the user that initialize the client(worker process) has appropriate permissions. Patch subscription is needed in the backend for dedup.

File: openlcs/packages/permissions.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+class ReadOnlyModelPermission(BasePermission):
+    """
+    Wrapper permission control class, to restrict the API create/update
+    access only to admin/superuser.
+    """
+    def has_permission(self, request, view):
+        if request.method == 'GET':
+            return True
+        else:
+            return request.user.is_staff
```
[the remark this answers is not in the document]
Comment (rhyw) [reply openlcs/packages/permissions.py:13]: I prefer `is_staff`,  you could check its doc.
`is_superuser`to me is like granting root while shelling a server that can do anything.

In our case, admin users will always be `is_staff=True`.

[the remark this answers is not in the document]
Comment (rhyw) [reply openlcs/packages/permissions.py:13]: is_superuser works too of course.

[the remark this answers is not in the document]
Comment (hanchuntao) [reply openlcs/packages/permissions.py:13]: OK

**Comment by hanchuntao** [comment]
LGTM

**Pull request merged**